akka-remote/src/main/scala/akka/remote/artery/BufferPool.scala

```diff
@@ -318,14 +386,26 @@ private[remote] final class EnvelopeBuffer(val byteBuffer: ByteBuffer) {
     if ((refCompressionVersionTag & TagTypeMask) != 0) {
       header._inboundActorRefCompressionTableVersion = refCompressionVersionTag & TagValueMask
     }
-    val manifestCompressionVersionTag = byteBuffer.getInt(ClassManifestCompressionTableVersionTagOffset)
+    val manifestCompressionVersionTag = byteBuffer.getShort(ClassManifestCompressionTableVersionTagOffset)
```

Comment:
hm, shouldn't this be done in issue https://github.com/akka/akka/issues/20872 instead of starting to do it here without support for roll over?